src/SDKs/_metadata/datalake-store_resource-manager.txt

```diff
@@ -1,11 +1,11 @@
-2018-01-04 20:02:08 UTC
+2018-02-05 18:29:47 UTC
 
 1) azure-rest-api-specs repository information
-GitHub user: Azure
-Branch:      current
-Commit:      db22f73d75fba57c9f1dcbd118bd1337ade842cf
+GitHub user: ro-joowan
+Branch:      revert-path-parameters
```

Comment:
this has to be generated from master branch

Reply:
@shahabhijeet Done.  I also updated the metadata files for ADLA -- the PR that you already merged: https://github.com/Azure/azure-sdk-for-net/pull/4043